back-end/handlers/hostHelpers.ts

```diff
@@ -164,30 +181,35 @@ const hostShowAnswer = async (gameId: number, io: Server): Promise<void> => {
 
   const players = await playerDb.getPlayers(gameId);
   const subjectPlayerId = gameData.quizQuestions[currentQuestionIndex].playerId;
-  const guessingPlayers = players.filter(p => p.id !== subjectPlayerId);
+
+  const guessingPlayers = players.filter(p => p.id !== subjectPlayerId && p.quizGuesses[currentQuestionIndex]);
   const guesses = await playerDb.getPlayerGuessesForQuizQuestion(gameId, currentQuestionIndex);
 
   const correctGuess = gameData.quizQuestions[currentQuestionIndex].correctAnswerIndex;
   const totalGuesses = guesses.length - 1;
-  const numCorrect = guesses.filter(g => g &&g.guess === correctGuess).length;
+  const numCorrect = guesses.filter(g => g && g.guess === correctGuess).length;
 
-  if (numCorrect === 0) {
+  if (subjectPlayerId === "friends") {
+    await handleTiebreakerAnswers(players, currentQuestionIndex, correctGuess);
+  } else if (numCorrect === 0) {
     playerDb.awardAllPlayersConsolationPoints(guessingPlayers, currentQuestionIndex);
   }
 
   const subjectPlayer = await playerDb.getPlayer(subjectPlayerId);
   const subjectBonus = Math.floor(300 * (numCorrect / totalGuesses));
 
-  await Player.updateOne({
-    id: subjectPlayer.id
-  }, { 
-    $set: {
-      'score': subjectPlayer.score + subjectBonus
-    }
-  });
+  if (subjectPlayer) {
```

Comment:
only case when there's not a `subjectPlayer` is the tiebreaker - this handles that